src/cedar.js

```diff
@@ -744,9 +744,9 @@ Cedar._validateData = function(data, mappings){
  */
 Cedar._getMappingFieldName = function(mappingName, fieldName){
   var name = fieldName;
-  //if(mappingName.toLowerCase() === 'count'){
-  //  name = fieldName + '_SUM';
-  //}
+  if(mappingName.toLowerCase() === 'count'){
```

Comment:
Since purpose of this PR is to fix broken tests, it concerns me a  bit to see logic changes to the source. I haven't looked into this at all, but someone had a reason to comment out these lines. Best to look up the history w/ blame and if indeed you do need to make this change, please document why,